jira/client.py

```diff
@@ -2638,6 +2639,7 @@ def my_permissions(
             projectId (Optional[str]): limit returned permissions to the specified project
             issueKey (Optional[str]): limit returned permissions to the specified issue
             issueId (Optional[str]): limit returned permissions to the specified issue
+            permissions (Optional[str]): limit returned permissions to the specified csv permission keys (cloud required field)
```

Comment:
Is there a link that we can reference to help users find out what keys are available?

Reply:
we can also raise an exception, e.g. `ValueError` if the user does not pass this variable but is running a cloud instance

Reply:
[This](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-permission-schemes/#api-rest-api-3-permissionscheme-get ) page documents collections of permissions. I noticed also that the admin user created in ```make_local_jira_user.py``` contains 73 permissions which is much more than the ones documented in that page. What would be your suggestion in terms of improving docstring to include some Jira doc reference?

Reply:
> we can also raise an exception, e.g. `ValueError` if the user does not pass this variable but is running a cloud instance

At the moment, if the user does not provide ```permissions``` field on a query for cloud Jira, it will raise a ```JiraError``` (with message ```"The 'permissions' query parameter is required."```). Is ```ValueError``` better than ```JiraError```? I personally prefer a ```JiraError``` but better be consistent in case other required fields are raising ```ValueError``` when not specified.

Reply:
Ah nice, I think that existing `JiraError` is fine as is. It would make sense for us to handle it if the message was confusing, but that sounds very clear. But we should perhaps add a test for that case, ie we do not pass the parameter and we get the exception raised.

In terms of how to reference the docs, I would just add the full url, it looks like that is the standard elsewhere in other docstrings.

Reply:
For now I think that we want to cover the Cloud case with the extra documentation due to the API change.

I think specifically we want to hyperlink:
https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-permission-schemes/#built-in-permissions

Reply:
Gotcha ! I added commit [7f82941](https://github.com/pycontribs/jira/pull/1472/commits/7f82941c8539196ad45f3a9f4e153539e2e79da2) that hopefully improves it but feel free to improve on it.